.github/workflows/build.yml

```diff
@@ -204,7 +204,9 @@ jobs:
       - name: Sanity check fuzz targets on Rust ${{ env.TOOLCHAIN }}
         run: cd fuzz && RUSTFLAGS="--cfg=fuzzing" cargo test --verbose --color always
       - name: Run fuzzers
-        run: cd fuzz && ./ci-fuzz.sh
+        run: cd fuzz && ./ci-fuzz.sh && cd ..
+      - name: Run lightning-invoice fuzzers
+        run: cd lightning-invoice/fuzz && cargo test --verbose && ./ci-fuzz.sh
```

Comment:
We should probably leave `cfg=fuzzing` here in the test call just because - underlying rust libs will be using it soon and we'll want to swap off of the fuzztarget feature entirely soon, I think.